ludwig/utils/strings_utils.py

```diff
@@ -34,7 +34,7 @@
 PANDAS_FALSE_STRS = {"false"}
 
 BOOL_TRUE_STRS = {"yes", "y", "true", "t", "1", "1.0"}
-BOOL_FALSE_STRS = {"no", "n", "false", "f", "0", "0.0"}
+BOOL_FALSE_STRS = {"no", "n", "false", "f", "0", "0.0", "-1"}
```

Comment:
Should we also throw in `-1.0` for consistency with 0 and 1?

Reply:
Added